service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -277,6 +277,14 @@ public String getOrderAppliesToAllChildren() {
         return orderCollection != null ? orderCollection : new ArrayList<>();
     }
 
+    @JsonIgnore
+    public Optional<GeneratedOrder> getOrderFromUUID(UUID elementId) {
```

Comment:
We don't need this method, you can use the one provided by element utils instead

Reply:
or if you still want it it can be simplified by using the method stated above